slu/slu/constants/__init__.py

```diff
@@ -130,8 +132,15 @@
 CURRENT_INTENT = "current_intent"
 STATE = "state"
 EXPECTED_SLOTS = "expected_slots"
+NLS_LABEL = "nls_label"
+PROMPT_NOISE_FILLER_TOKEN = "<pad>"
 
-LANG_TO_LOCALES = {"en": "en_IN", "hi": "hi_IN"}  # This should be set via config
+LANG_TO_LOCALES = {                     
+    "en": "en_IN", 
+    "hi": "hi_IN", 
+    "hi": "hi_nls", 
+    "en": "en_nls"
+}                                        # This should be set via config
```

Comment:
Wait, how can a dictionary have duplicate keys?

Reply:
Changed this, added a new constant instead